.pre-commit-config.yaml

```diff
@@ -106,3 +106,10 @@ repos:
         entry: ./activated.py flake8
         language: system
         types: [python]
+  - repo: local
+    hooks:
+      - id: ruff
+        name: Ruff
+        entry: ./activated.py ruff check --fix
+        language: system
+        pass_filenames: false
```

Comment:
can we still let pre-commit handle the filenames?  i guess ruff is faster so maybe we can just inefficiently check everything, but...  `:]`

Reply:
done